tests/foreman/api/test_hostgroup.py

```diff
@@ -389,7 +385,7 @@ def test_positive_create_with_ptable(self):
         ).create()
         self.assertEqual(hostgroup.ptable.read().name, ptable.name)
 
-    @tier1
+    @tier2
```

Comment:
can we not keep as `tier1` as it's very basic functionality of component ? what do you think ?

same for below changes where it changed ..